ivy.xml

```diff
@@ -9,36 +9,6 @@ SPDX-License-Identifier: AGPL-3.0-only
 <ivy-module version="2.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://ant.apache.org/ivy/schemas/ivy.xsd">
```

Comment:
Is this file still required?

Reply:
I removed it, it is not required, thanks